server/dash/dashboards/neighborhood_recent.py

```diff
@@ -149,12 +159,19 @@ def populate_options():
     Input("council_list", "value")
 )
 def update_table(selected_council):
-    """Dash Callback Function to update the LA 311 request data table at the bottom of the dashboard.
+    """Filters the LA 311 request data table based on selected_council.
 
     This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
     outputs a list of requests associated with that nc as a data table in dictionary form 
     with id "council_table" in the layout. 
 
+    Args:
+        selected_council:
```

Comment:
nit: please put next line on the same line as this one. I.e.,
`selected_council: A string...`

Same for the rest of the docstrings.

Reply:
Resolved.